packages/browser/src/client/tester/utils.ts

```diff
@@ -167,11 +167,14 @@ export function processTimeoutOptions<T extends { timeout?: number }>(options_?:
 }
 
 export function getIframeScale(): number {
-  const testerUi = window.parent.document.querySelector('#tester-ui') as HTMLElement | null
+  const testerUi = window.parent.document.querySelector(`iframe[data-vitest]`)?.parentElement
   if (!testerUi) {
     throw new Error(`Cannot find Tester element. This is a bug in Vitest. Please, open a new issue with reproduction.`)
   }
   const scaleAttribute = testerUi.getAttribute('data-scale')
+  if (scaleAttribute === null) {
```

Comment:
We should always add a `data-scale` attribute and throw an error if it's incorrect. The `isNaN` check after that checks exactly that scenario